nextflow_schema.json

```diff
@@ -15,7 +15,7 @@
                     "required": ["submission_basepath"]
                 },
                 {
-                    "required": ["input", "genome"]
+                    "required": ["input"]
```

Comment:
why did you delete genome there? if input is samplesheet genome is required to run the pipeline. Did you change the logic?